v1/envs_set_up.sh

```
@@ -0,0 +1,51 @@
+#!/bin/zsh
```

Shellcheck (0.10.0)

[error] ShellCheck only supports sh/bash/dash/ksh/'busybox sh' scripts. Sorry!

(SC1071)